src/secret_sharing/replicated/malicious/additive_share.rs

```diff
@@ -275,6 +303,19 @@ mod tests {
         test_fixture::Reconstruct,
     };
 
+    // The field Fp31 is *not* large enough to provide statistical security. As such, it should never
+    // be used (in production) for malicious circuits.
+    // That said, it's still very useful for this to work in *tests*. One reason is that it's useful to have
+    // a small field where collissions can often happen.
+    #[cfg(test)]
+    impl ExtendableField for Fp31 {
+        type LargeFieldType = Fp31;
+
+        fn get_induced_value(&self) -> Self::LargeFieldType {
+            *self
+        }
+    }
```

Comment:
As noted, I think that we can move this annotation to the declaration of `Fp31` itself.

Reply:
I'm still a bit confused about where you'd recommend I move this to. To PrimeField? In this file, or another file? And where should I put the #[cfg(test)]?